packages/compartment-mapper/src/import-archive.js

```diff
@@ -151,7 +151,7 @@ const makeArchiveImportHookMaker = (
         sourceLocation,
         packageLocation,
       );
-      return record;
+      return { record, specifier: moduleSpecifier, importMeta: { url: null } };
```

Comment:
make it a 
```suggestion
      return { record, specifier: moduleSpecifier, importMeta: {  } };
```

Reply:
Or rather, omit the `importMeta` property entirely.